subprocess/subtree_oom.go

```diff
@@ -63,3 +68,58 @@ func getPidFromLog(line string) (int, bool) {
 	}
 	return pid, true
 }
+
+func ClearOOMHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		grip.Debug("clearing oom")
```

Comment:
remove this debug line, it just goes to the local file and is probably a bit confusing